cmd/kube-apiserver/app/options/validation.go

```diff
@@ -45,6 +45,20 @@ func validateServiceNodePort(options *ServerRunOptions) []error {
 	return errors
 }
 
+func validateAuditLogFlags(options *ServerRunOptions) []error {
```

Comment:
Packages in the `staging` directory are symlinked to vendor so the code will compile. https://github.com/kubernetes/kubernetes/tree/master/staging

For any package in `staging` , like `k8s.io/apiserver`, you should edit the `vendor` directory directly. All other packages in `vendor` should be changed in their canonical repo, then updated using godep.